pkg/helmshim/helm_test.go

```diff
@@ -7,8 +7,45 @@ import (
 	"github.com/Azure/eno/pkg/function"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
+func TestIsNullObject(t *testing.T) {
+	cases := []struct {
+		name string
+		o    *unstructured.Unstructured
+		want bool
+	}{
+		{
+			name: "nil object",
+			o:    nil,
+			want: true,
+		},
+		{
+			name: "empty object",
+			o:    &unstructured.Unstructured{},
+			want: true,
+		},
+		{
+			name: "non-empty object",
+			o: &unstructured.Unstructured{
+				Object: map[string]any{
```

Comment:
Needs another test case to cover `o.Object = map[string]any{}`

Reply:
technically helm should not return value like this because it's also checking on gvk if the yaml file is not empty. Added a validation logic on this too, but this will result in we have to use json.Marshal to generate the encoded result.